recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "asteval" %}
-{% set version = "0.9.16" %}
-{% set sha256 = "bc71bb4f9ad3ef8ccdc3b9a711e094fb00d768172cb7ff9a15773673aac9e0cc" %}
+{% set version = "0.9.21" %}
+{% set sha256 = "ee14ba2211cda1c76114e3e7b552cdd57e940309203d5f4106e6d6f2c2346a2e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/4f/d5/ce490afa6f6ab1bb97ffd66a80bfb032d22407fc215d60b21ca174e25c51/asteval-0.9.21.tar.gz
```

Comment:
Please leave the jinja2 in if possible. Did the existing jinja2 not work?

Reply:
@CJ-Wright  restored to better URL. I swear that when I tried that with `curl` it failed, but it did just work for me from a browser.